daemon/mgr/spec_process.go

```diff
@@ -59,27 +58,20 @@ func setupProcessTTY(ctx context.Context, c *ContainerMeta, spec *SpecWrapper) e
 }
 
 func setupProcessUser(ctx context.Context, c *ContainerMeta, spec *SpecWrapper) (err error) {
+	// The user option is complicated, now we only handle case "uid".
+	// TODO: handle other cases like "user", "uid:gid", etc.
 	if c.Config.User != "" {
-		fields := strings.SplitN(c.Config.User, ":", 2)
-		var u, g string
+		fields := strings.Split(c.Config.User, ":")
+		var u string
 		u = fields[0]
```

Comment:
u still has possibility of being empty string, just make `c.Config.User` to be `:asdfghj`. right?